action.yml

```diff
@@ -228,11 +228,11 @@ runs:
       id: set-output
       run: |
         echo "::group::Set version output"
-        echo "::set-output name=kubeconfig::$HOME/.kube/config"
-        echo "::set-output name=k3s-version::$(k3s --version | sed 's/.*\(v[0-9][^ ]*\).*/\1/')"
-        echo "::set-output name=k8s-version::$(k3s --version | sed 's/.*\(v[0-9][^+]*\).*/\1/')"
-        echo "::set-output name=calico-version::$(cat /tmp/calico.yaml | grep --max-count=1 "calico/cni:v" | sed 's/.*calico\/cni:\(.*\)/\1/')"
-        echo "::set-output name=helm-version::$(helm version --short | sed 's/\([^+]*\).*/\1/')"
+        echo "kubeconfig=$HOME/.kube/config" >> $GITHUB_OUTPUT
+        echo "k3s_version=$(k3s --version | grep --max-count=1 'k3s' | sed 's/.*\(v[0-9][^ ]*\).*/\1/')" >> $GITHUB_OUTPUT
```

Comment:
This introduces a mismatch between the output `k3s_version`, input `k3s-version` (probably can't do anything about this since it'll break everything), and the CI test in this repo:
https://github.com/jupyterhub/action-k3s-helm/blob/1039e0fe18752d4aa4a2f8594382d343c1f9728e/.github/workflows/test_k3s.yml#L81-L84

What happens if you keep the `-` in the env var name? It's not supported by bash, but if this is all internal to GitHub workflows it might work: https://unix.stackexchange.com/a/23714

Reply:
Lets try!

Reply:
It worked! :D Okay!